web/app/admin/views.py

```diff
@@ -53,6 +53,21 @@ def add_teacher():
     return render_template('admin/add.html', form=form)
 
 
+@admin.route('/teachers/edit/<int:id>', methods=['GET', 'POST'])
+@login_required
+@admin_required
+def edit_teacher(id):
+    user = User.query.filter_by(id=id).first()
+    form = EditTeacherForm(obj=user)
```

Comment:
I guess this is the line that populate the form with the current data. Where should we force the form into displaying the user's email twice?